internal/pkg/cli/pipeline_delete.go

```diff
@@ -124,8 +108,31 @@ func (o *deletePipelineOpts) Validate() error {
 		return errNoProjectInWorkspace
 	}
 
-	if o.PipelineName == "" {
-		return errNoPipelineInWorkspace
+	if err := o.readPipelineManifest(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (o *deletePipelineOpts) readPipelineManifest() error {
+	data, err := o.ws.ReadPipelineManifest()
+	if err != nil {
+		if _, ok := err.(*workspace.ErrNoPipelineInWorkspace); ok {
+			return err
+		}
+		return fmt.Errorf("read pipeline manifest: %w", err)
```

Comment:
I believe we can just `return fmt.Errorf("read pipeline manifest: %w", err)` because of the "%w" verb.

Then in `deleteProjectPipeline` in `project_delete.go` we can catch a wrapped error with:
```go
if !errors.Is(err,  workspace.ErrNoPipelineInWorkspace) {
	return err
}
```

Reply:
getting strange behavior in the test so leaving this for now.